understanding/22/target-size-minimum.html

```diff
@@ -104,49 +104,53 @@ <h3>Size requirement</h3>
+          <p>The ability to select the smaller button is reduced in the second version, as it is harder to successfully hit and there is little spacing around the target. In the third version the target is still harder to hit, but it is less likely that you'd hit a neighboring target. The measure for target offset works intuitively for most cases, but in some cases, at small sizes, it can cause additional spacing to be required.</p>
+
+          <p>As shown in the previous illustrations, it is still possible to meet this Success Criterion when targets are not 24 by 24 CSS pixels, provided that there is sufficient spacing. The next two illustrations show sets of buttons which are only 20 CSS pixels tall. In the first set, there are no targets immediately above or below the buttons, so they pass. In the second illustration, the buttons have been stacked on top of one another, resulting in a fail.</p>
           <figure id="pointer-target-44x20-spacing">
-              <img src="img/pointer-target-example3-v2.png" alt="A row of buttons which are more that 44px wide and 20 CSS pixels high. There are no targets above or below." />
-              <figcaption>The widths of the buttons with adjacent neighbors is above 24 CSS pixels, while the height is only 20 CSS pixels. However, the lack of adjacent targets above and below means that the targets pass this Success Criterion.</figcaption>
+            <img src="img/pointer-target-example3-v2.png" alt="A row of buttons which are more that 44px wide and 20 CSS pixels high. There are no targets above or below." />
+            <figcaption>The widths of the buttons with adjacent neighbors is above 24 CSS pixels, while the height is only 20 CSS pixels. However, the lack of adjacent targets above and below means that the targets pass this Success Criterion.</figcaption>
           </figure>
 
           <figure id="pointer-target-44-20-nospace">
-              <img src="img/pointer-target-example4-v2.png" alt="Two rows of buttons only 20 CSS pixels high, with no spacing between them." />
-              <figcaption><strong>Fail:</strong> Two rows of buttons, both with a height of only 20 CSS pixels. The rows are flush, with no gap between them, which means that the offset from the top of the targets in the top row to the top of targets in the row below is below 24px, thus failing the Success Criterion.</figcaption>
+            <img src="img/pointer-target-example4-v2.png" alt="Two rows of buttons only 20 CSS pixels high, with no spacing between them." />
+            <figcaption><strong>Fail:</strong> Two rows of buttons, both with a height of only 20 CSS pixels. The rows are flush, with no gap between them, which means that the offset from the top of the targets in the top row to the top of targets in the row below is below 24px, thus failing the Success Criterion.</figcaption>
```

Comment:
I noticed that all the "Fail" and "Pass" text at the start of the figcaptions were removed in this PR, except this one. So I thought I'd remove for consistency.
```suggestion
            <figcaption>Two rows of buttons, both with a height of only 20 CSS pixels. The rows are flush, with no gap between them, which means that the offset from the top of the targets in the top row to the top of targets in the row below is below 24px, thus failing the Success Criterion.</figcaption>
```

Reply:
this is part of my messed-up merge i think. i might have to do a fresh re-do of this PR then force-push